deviceModeInit.js

```diff
@@ -356,10 +388,15 @@ var rudderTracking = (function () {
     };
     if (pages[path] === "Registration Viewed") {
       rudderanalytics.track(pages[path], properties);
-      rs$("#create_customer").submit(userRegistered);
+      if (!disableClientIdentifierEvents) {
```

Comment:
Rather than calling `disableClientIdentifierEvents` can we call it `enableClientIdentifierEvents`  so that we don't to put a ! before each `if`

Reply:
Done